### PR TITLE
Remove Velero-plugin-for-csi from operator csv

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**

### CODE REVIEW

**Comment by mateusoliveira43** [comment]
does this file `deploy/disconnected-prep.sh` need update as well?

**Comment by shubham-pampattiwar** [comment]
> does this file `deploy/disconnected-prep.sh` need update as well?

We usually don't use the `deploy/disconnected.sh` script, downstream uses it and it resides downstream. But I still updated the script for reference purposes.

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -856,8 +856,6 @@ spec:
                 - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_GCP
                   value: quay.io/konveyor/velero-plugin-for-gcp:oadp-1.4
                 - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_CSI
-                  value: quay.io/konveyor/velero-plugin-for-csi:oadp-1.3
-                - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
```
Comment (sseago) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:860]: Looks like the wrong name was removed. 858 above should be removed, this one should stay.

**Review by sseago — CHANGES_REQUESTED** [review]

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (rayfordj) [reply bundle/manifests/oadp-operator.clusterserviceversion.yaml:860]: Agreed; wrong line removed. :sunglasses: 

FWIW, `make bundle` appears to do the correct thing when I run it locally

```
-                - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_CSI
+                - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
```

Comment (rayfordj) [reply bundle/manifests/oadp-operator.clusterserviceversion.yaml:860]: Resolved in https://github.com/openshift/oadp-operator/pull/1407/commits/bfc3f5e75954999bad1067af7f165177fd01cdbf

**Review by rayfordj — APPROVED** [review]
/lgtm

**Pull request merged**